### DISCUSSION TITLE
질문을 어떻게 하는 지 모르겠어요.

### DISCUSSION THREAD

**Discussion by durume** [fusiondevkr/pl900#3] [1 Q&A] [answered]
정작 질문하는 방법을 모를까봐 혼자서 묻고 답을 해봅니다.

**Comment by durume** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/83335854/125952394-806ef9b0-9a80-4e1f-8371-4ffe49f63785.png)
1번과 같이 질의응답 게시판(화면에서는 'Discussions')로 이동하고,
2번의 New Discussion을 눌러서 카테고리 선택후 글을 쓰시면 됩니다.

**Comment by justinyoo** [comment]
그리고 여러분들이 답을 달아 주실텐데, 그 중에 질문하신 분께서 가장 내가 원하는 답이다! 라고 생각하신다면 해당 답의 우측 상단에 ☑️ 버튼을 눌러 답으로 만들어 주시면 아주 좋습니다!

![image](https://user-images.githubusercontent.com/1538528/125968980-61c98198-991d-48a2-bc70-a62e3910d564.png)
